tcp/pcb.ml

```diff
@@ -479,7 +486,13 @@ struct
         (* Finish processing ACK, so pcb.state is correct *)
         Rx.input t pkt newconn >>= fun () ->
         (* send new connection up to listener *)
-        pushf (fst newconn)
+        let flow = fst newconn in
+        Lwt.catch (fun () -> pushf flow)
```

Comment:
Should we add a `Lwt.async` here too? I'm not sure, but it looks like we run the entire TCP flow handler in the context of the original network frame. We should probably return sooner, to let netif know we're done with the ack frame.